modules/search-service-impl/src/main/java/org/opencastproject/search/impl/persistence/SearchServiceDatabase.java

```diff
@@ -164,9 +164,11 @@ Collection<AccessControlList> getAccessControlLists(String seriesId, String ...
    *           if exception occurs
    * @throws NotFoundException
    *           if media package with specified id is not found
+   * @throws UnauthorizedException
+   *           if the current user is not authorized to perform this action
    */
   void deleteMediaPackage(String mediaPackageId, Date deletionDate) throws SearchServiceDatabaseException,
-          NotFoundException;
+          NotFoundException, UnauthorizedException;
```

Comment:
I added some (potentially) missing exceptions. Other methods in this class are using these exceptions, and it seems they were simply overlooked.